dts/arm/nxp/nxp_imx.dtsi

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright (c) 2017, NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <arm/armv7-m.dtsi>
+#include <dt-bindings/clock/imx_ccm.h>
```

Comment:
This looks unused. The header is related to `drivers/clock_control/clock_control_mcux_ccm.c`, which has some minimum implementation to support iMX RT1050 (which uses `ext/hal/nxp/mcux`). The UART shim driver uses function from `clock_freq.c`.

Reply:
I'll fix this.